app/controllers/declaration/TotalNumberOfItemsController.scala

```diff
@@ -53,21 +53,21 @@ class TotalNumberOfItemsController @Inject()(
   }
 
   def saveNoOfItems(mode: Mode): Action[AnyContent] = (authenticate andThen journeyType(validTypes)).async { implicit request =>
-    form.bindFromRequest
-      .fold(
-        formWithErrors => Future.successful(BadRequest(totalNumberOfItemsPage(mode, formWithErrors))),
-        updateCache(_).map(_ => navigator.continueTo(mode, TotalPackageQuantityController.displayPage))
-      )
+    form().bindFromRequest
+      .fold(formWithErrors => {
+        Future.successful(BadRequest(totalNumberOfItemsPage(mode, formWithErrors)))
+      }, updateCache(_).map(_ => navigator.continueTo(mode, TotalPackageQuantityController.displayPage)))
```

Comment:
IMHO this syntax change just add noise. The former was cleaner and more readable, though I understand this is subjective. But if I may... why the braces?

Reply:
@darrylmurphy Do you want to comment here? I can revert if wanted.